datadog/resource_datadog_dashboard.go

```diff
@@ -3802,6 +3802,7 @@ func getServiceLevelObjectiveDefinitionSchema() map[string]*schema.Schema {
 		"slo_id": {
 			Type:     schema.TypeString,
 			Required: true,
+			ForceNew: true,
```

Comment:
Will this cause the whole dashboard to be deleted + recreated if the slo changes? I'm not sure if that'll be expected for users, since it'd mean any dashboard lists or favorited dashboards will need to be updated with the new dashboard id.

Reply:
I don't know if it's an issue any more: the dasboard_list can now be handle by the dashboard itself (see: https://github.com/DataDog/terraform-provider-datadog/pull/654/files) so in theory it should works just fine. But I' m agree, delete + recreate could be a little ... overkill :smile: